invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentRequestDetails/components/RequestActions/RequestActions.js

```diff
@@ -14,9 +13,8 @@ export default class RequestActions extends Component {
 
   getAvailableActions() {
     const { pid, state } = this.props.documentRequestDetails.metadata;
-    const actions = [];
 
-    if (state == 'PENDING') {
+    if (state === 'PENDING') {
```

Comment:
I'm a bit concerned to have the same state strings for the circulation and the document requests. They should be probably in the invenioConfig with separate names. @ntarocco WDYT?